specification/app/resource-manager/Microsoft.App/preview/2023-04-01-preview/CommonDefinitions.json

```diff
@@ -676,6 +680,10 @@
         "mountPath": {
           "description": "Path within the container at which the volume should be mounted.Must not contain ':'.",
           "type": "string"
+        },
+        "subPath": {
+          "description": "Path within the volume from which the container's volume should be mounted. Defaults to '' (volume's root).",
+          "type": "string"
```

Comment:
should it have a `"default": ""`?

Reply:
> should it have a `"default": ""`?

No need, null is acceptable.

Reply:
`default` is server default

x-ms-client-default is client default, not asking for this.